File: website/client/components/language/kotlin/tachyon-wasm.kt
var count: Int = 6
val label: String = "from Kotlin"

fun doubled(): Int = count * 2

val tac =
    mapOf(
        "count" to TacField({ count }, { count = tacInt(it) }),
        "label" to TacField({ label }),
        "doubled" to TacMethod { doubled() },
    )

File: website/client/components/language/dart/tachyon-wasm.dart
int count = 6;
String label = 'from Dart';

int doubled() => count * 2;

final tac = {
  'count': TacField(() => count, (value) => count = value as int),
  'label': TacField(() => label),
  'doubled': TacMethod((arguments) => doubled()),
};
